README.md

```diff
@@ -90,6 +92,31 @@ require('tasks').setup({
 
 Cargo module doesn't have a `target` param which specific to CMake because `cargo run` automatically pick the binary. If there is multiple binaries, you can set which one you want to run using `--bin` or `--project` in step 2 as you do in CLI.
 
+### GNU Make
+
+1. Open a Make project.
+1. Run a Make target `<target>` with `:Task start make <target>`.
```

Comment:
I know that Markdown automatically numerates lists, but above I used manual numeration. Could you do it as above just for consistency?
```suggestion
2. Run a Make target `<target>` with `:Task start make <target>`.
```